students/mkdir01/lesson03/list_lab.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+#SERIES 1
+
+# Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+
+# Display the list (plain old print() is fine…).
+print(fruits)
+
+# Ask the user for another fruit and add it to the end of the list.
+response_fruits = input("Type another fruit: ")
+fruits.append(response_fruits)
+
+# Display the list.
+print(fruits)
+
+# Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis). Remember that Python uses zero-based indexing, so you will need to correct.
+response_num = input("Type a number: ")
+print(response_num)
+print(fruits[int(response_num) - 1])
+
+
+# Add another fruit to the beginning of the list using “+” and display the list.
+fruits = ["Berries"] + fruits[:]
+print(fruits)
+
+# Add another fruit to the beginning of the list using insert() and display the list.
+fruits.insert(0, "Plums")
+print(fruits)
+
+# Display all the fruits that begin with “P”, using a for loop.
+for i in range(len(fruits)):
+    if "P" in fruits[i]:
```

Comment:
This should use `str.startswith()` or `0` item index to check the first letter in the word